docs/src/tutorials/getting_started/getting_started_with_sets_and_indexing.jl

```diff
@@ -116,6 +116,35 @@ x["A", "D"]
 # !!! info
 #     Note how we indexed `x["A", "D"]` instead of `x[("A", "D")]` as above.
 
+# ## Sets to watch out for
+
+# JuMP supports any sets which are iterable, that is, the set `set` supports
+# a for-loop like: `[i for i in set]`. This causes a few common errors.
+
+# First, if `T = 3`, you may pass the integer `T` instead of the range `1:T`:
```

Comment:
"instead of range" -> "and index a single variable instead of indexing over the intended range `1:T`:"